docs/content/latest/yugabyte-platform/security/create-kms-config.md/aws-kms.md

```diff
@@ -30,9 +30,16 @@ showAsideToc: true
     </a>
   </li>
 
+  <li >
+    <a href="{{< relref "./hashicorp-kms.md" >}}" class="nav-link">
+      <i class="icon-postgres" aria-hidden="true"></i>
+      HashiCorp Vault
```

Comment:
Hashicorp

Reply:
HashiCorp is correct (see https://www.hashicorp.com/products/vault )